appserver/packager/legal/src/main/resources/glassfish/legal/3RD-PARTY-LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-GlassFish Server_4.0_THIRDPARTYLICENSEREADME.txt
+GlassFish Server_5.0_THIRDPARTYLICENSEREADME.txt
```

Comment:
Please change this line to look like this:
GlassFish Server 5.0 3RD-PARTY-LICENSE.txt